numpy_financial/tests/test_financial.py

```diff
@@ -421,20 +421,20 @@ def test_rate_nan(self, number_type):
         Both payments and present value are positive, it is impossible to pay off the existing balance
         by making further withdrawals, regardless of the rate.
         """
-        rate = numpy.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(0))
+        rate = npf.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(0))
         assert_equal(numpy.nan, float(rate))
-        rate = numpy.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(5000))
+        rate = npf.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(5000))
         assert_equal(numpy.nan, float(rate))
 
         # begin
-        rate = numpy.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(20000), 1)
+        rate = npf.rate(number_type(12.0), number_type('400.0'), number_type('10000.0'), number_type(20000), 1)
```

Comment:
> nit: using number_type(12.0) vs number_type('-400.0') is a little inconsistent. Can you just use number_type(-400) etc in all places here? -- @eric-wieser from the original PR

Would you please update this?

Reply:
@eric-wieser @Kai-Striega Changes done as per above comment. Please let me know if you need any other change as well.

Reply:
These now look good to me.